### PR TITLE
Separate pile and preview bg color

### PR DESCRIPTION
**Pull request by eeeeenchanted**
## Description

> What was changed in this pull request?

Added `previewBorderColor` and `previewBorderOpacity` to the Redux store.

Now the `previewBackgroundColor` is used when the preview is created, and the `previewBorderColor` is used when hovering on the preview.

> Why is it necessary?

To separate between the pile bg color and the preview bg color.

## Checklist

- [x] GitHub labels properly set (e.g., bug, new feature, etc.)
- [x] Documentation added or updated
- [ ] Examples added or updated
- [ ] Screenshot or GIF included (e.g. new or changed visualization features)
- [x] CHANGELOG.md updated

### CODE REVIEW

**Comment by eeeeenchanted** [comment]
Well, the preview bg is created together with the preview. Previously `pileBackgroundColor` defines the preview bg color, that's why when you changed `pileBackgroundColor` to white, you would see the preview background although you didn't hover on the pile. Now the `previewBackgroundColor` is used when not interact with the preview, and `previewBorderColor` is used when hovering on the preview.

Hmm... now I think the preview bg color should just be `null` when creating the preview, then the preview bg will only be drawn when hovering on the preview.

**Comment by flekschas** [comment]
> Previously `pileBackgroundColor` defines the preview bg color, that's why when you changed `pileBackgroundColor` to white, you would see the preview background although you didn't hover on the pile.

I actually think that's fine. This is not an issue of piling.js but rather an issue of how it's configured. If you set `previewBackgroundColor` to `#ffffff` one will run into the same issue.

Or am I missing something?

> Hmm... now I think the preview bg color should just be null when creating the preview, then the preview bg will only be drawn when hovering on the preview.

I think it's good that the previews have a distinct background color. This makes them appear as one cohesive object rather than individual objects. But I agree there might be times where one doesn't want a background color so `null` should refer to transparent.

In conclusion, I think it's good to have the new properties but I suggest that  `previewBackgroundColor` by default inherits the color from `pileBackgroundColor` to make things less verbose. We could define a special property called `inherit` which is set by default and makes the preview bg color to inherit from the pile.

What do you think?
